modules/client-configuration/pages/registration-bootstrap.adoc

```diff
@@ -121,6 +121,11 @@ ORG_GPG_KEY=
 
 . When you have finished, save the file, and repeat this procedure for the second bootstrap script.
 
+[NOTE]
+====
+By default bootstrap script will try to install [package]``venv-salt-minion`` for Salt clients if it's available in the bootstrap repository and [package]``salt-minion`` if there is no Salt bundle in the bootstrap repository.
+It is posible to avoid installing Salt bundle and keep using [package]``salt-minion`` if you need it for some reason. See xref:client-configuration:contact-methods-saltbundle.adoc[Salt Bundle] for details.
```

Comment:
This sentance is mostly about the bootstrapping process so without mentioning installation it looks weird as we will continue use the salt-minion anyway.

Reply:
It is possible to keep using [package]``salt-minion`` instead of the Salt Bundle if needed.

Reply:
In this case the meaning of the sentance distorted a bit. The full meaning of this sentance that it's possible to avoid using Salt Bundle and use salt-minion package as usual if needed. But the sentance makes sense onlt for bootstrapping process, for the systems which was registered already nothing is changing, such systems will keep using salt-minion anyway.